py/makeqstrdefs.py

```diff
@@ -20,6 +21,11 @@
 # Extract MP_COMPRESSED_ROM_TEXT("") macros.  (Which come from MP_ERROR_TEXT)
 _MODE_COMPRESS = "compress"
 
+# How many concurrent pre-processor instances to run to extract QSTRs.
+_PREPROCESS_PARALLELISM = 8
```

Comment:
Should we make this depend on number of CPUs? Something like...

    _PREPROCESS_PARALLELISM  = multiprocessing.cpu_count() - 1

Reply:
Done